lib/Db/Message.php

```diff
@@ -286,6 +293,27 @@ public function setFlag(string $flag, bool $value = true) {
 			);
 		}
 	}
+	/**
+	 * @param Avatar|null $avatar
+	 * @return void
+	 */
+	public function setAvatar(?Avatar $avatar): void {
+		$this->avatar = $avatar;
+	}
+
+	/**
+	 * @param bool $fetchAvatarFromClient
+	 */
+	public function setFetchAvatarFromClient(bool $fetchAvatarFromClient): void {
+		$this->fetchAvatarFromClient = $fetchAvatarFromClient;
+	}
+
+	/**
+	 * @return Avatar
```

Comment:
here the typehint contradicts the native one. Is the return value nullable or not? :wink: